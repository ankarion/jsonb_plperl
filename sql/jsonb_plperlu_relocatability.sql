CREATE EXTENSION jsonb_plperlu CASCADE;
CREATE SCHEMA test;
alter extension jsonb_plperlu set schema test;
create function test.test(val jsonb) returns jsonb
language plperlu
transform for type jsonb
as $$
return val
$$;

select test.test('1'::jsonb);

drop extension plperlu cascade;
drop schema test cascade;
