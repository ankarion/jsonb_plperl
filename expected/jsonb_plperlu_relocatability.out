CREATE EXTENSION jsonb_plperlu CASCADE;
NOTICE:  installing required extension "plperlu"
CREATE SCHEMA test;
alter extension jsonb_plperlu set schema test;
create function test.test(val jsonb) returns jsonb
language plperlu
transform for type jsonb
as $$
return val
$$;
select test.test('1'::jsonb);
 test  
-------
 "val"
(1 row)

drop extension plperlu cascade;
NOTICE:  drop cascades to 2 other objects
DETAIL:  drop cascades to extension jsonb_plperlu
drop cascades to function test.test(jsonb)
drop schema test cascade;
